### DISCUSSION TITLE
When to use "Discussions"?

### DISCUSSION THREAD

**Discussion by sojusnik** [excalidraw/excalidraw#2715] [General]
It's not obvious to me when to use the newly introduced "Discussions". The existing labels in "Issues" seem to overlap with the categories in "Discussions", namely:

[discussion](https://github.com/excalidraw/excalidraw/labels/discussion) with ["Discussions"](https://github.com/excalidraw/excalidraw/discussions)
[enhancement](https://github.com/excalidraw/excalidraw/labels/enhancement) with [Ideas](https://github.com/excalidraw/excalidraw/discussions/categories/ideas)
[support](https://github.com/excalidraw/excalidraw/labels/support) with [Q&A](https://github.com/excalidraw/excalidraw/discussions/categories/q-a)

At least, we should convert those labelled issues to "Discussions" in order to avoid further confusion and encourage the correct usage of "Discussions".

**Comment by dwelle** [comment]
It's not obvious to me either. I'd personally use only issues :stuck_out_tongue:. /cc @lipis

- `Discussion` is used for features which must be discussed with the team first before they are eligible for implementation. Thus, the label belongs to issues. It's comparable to `triage`/`TBD` or similar labels.
- `Enhancement` is definitely an issue — it should be considered either a `feature request`, or a `feature`. Since it's currently not possible to move discussions to issues, we wouldn't be even able to convert it to a trackable issue, later.
- `Support` — yea, that's a good fit for Discussions. Problem is, you never know when a support turns into a `bug` or `enhancement` etc., hence my preference for using Issues for everything :smile:

**Reply by sojusnik** [reply]
I would also convert or add a section for feature requests in "Discussions" and anticipate people to vote on proposals in there to easily sort for popular feature requests.

**Comment by lipis** [comment]
It was offered as an option and I wanted to test it out. Wasn't aware that those cannot become issues, and the whole thing is in Beta anyway.. I'm sure they are happy with some feedback and I'm positive that they will improve overtime.

> Since it's currently not possible to move discussions to issues, we wouldn't be even able to convert it to a trackable issue, later.

We can always open an issue and refer to the discussion for more info on why we need to do that.

Also as @dwelle mentions, Support is the best fit for discussions.. random stories.. and if something can be become a bug or a feature we can mark is as closed with the linked issue or PR.

Dunno.. I would give it more time until they are out of beta at least and provide feedback.. they really read those!

**Reply by sojusnik** [reply]
Yeah, "Discussions" will mature over time. I'm sure that they will implement the conversion from discussions to issues sooner or later.

**Comment by lipis** [comment]
Also you can accept answers.. It's the best place for QA/FAQ etc.. answers are usually pretty much lost in the issues.

**Reply by sojusnik** [reply]
Yes, Q&A and FAQ are a no-brainer for "Discussions".

**Comment by lipis** [comment]
https://github.com/github/feedback/discussions
